usaspending_api/accounts/models/appropriation_account_balances.py

```diff
@@ -56,6 +41,33 @@ class AppropriationAccountBalances(DataSourceTrackedModel):
     update_date = models.DateTimeField(auto_now=True, null=True)
     final_of_fy = models.BooleanField(blank=False, null=False, default=False, db_index=True)
 
+    class Meta:
+        abstract = True
+
+
+class AppropriationAccountBalancesManager(models.Manager):
+    def get_queryset(self):
+        """ Get only records from the last submission per TAS per fiscal year. """
+        return super(AppropriationAccountBalancesManager, self).get_queryset().filter(final_of_fy=True)
```

Comment:
This may look like new code because of refactoring ... but, did we always have this Manager class, and was it always defaulting to have a filter that forces `final_of_fy=True`. I thought we had plans to actually remove `final_of_fy` or stop using it on any A/B/C files. 

I'd be interested in what code was using this Model manager. Perhaps _that_ code is very aware of the filter and taking advantage of it.

Reply:
This is old code. I only see the `final_objects` referenced in one place (outside of definition). It could probably be removed, but I tried to limit changes in this PR.